etna/media/templatetags/media_tags.py

```diff
@@ -4,27 +4,29 @@
 
 
 @register.inclusion_tag("media/blocks/media-block--audio.html")
-def audio(media):
+def audio(title, media):
     """
     Inclusion tag to render a custom audio media block.
 
     Exposes "src" and "type" attributes for FE convenience.
     """
     return {
+        "title": title,
         "value": media,
         "src": media.sources[0]["src"],
         "type": media.sources[0]["type"],
     }
```

Comment:
I must say, I'm a bit confused about the need for these custom tags :S Why don't we just override `MediaBlock.get_context()` to include the `src` and `type` values, and `MediaBlock.get_template() to load the audio/video specific templates?

Reply:
I agree, I'm not sure why we're using them either - I didn't want to remove them though in case it was something we needed.
I'll do this instead! :)